stripe/src/main/java/com/stripe/android/view/IconTextInputLayout.java

```diff
@@ -0,0 +1,90 @@
+package com.stripe.android.view;
+
+import android.content.Context;
+import android.graphics.Rect;
+import android.support.annotation.VisibleForTesting;
+import android.support.design.widget.TextInputLayout;
+import android.util.AttributeSet;
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+/**
+ * This class uses Reflection to make the Support Library's floating hint text move above
```

Comment:
to make sure I understand correctly: 
The support library currently supports:
```
                [floating hint text]
[drawable left] [Edit text]
```

and we want
```
[floating text]
[drawable left] [edit text]
```
?

happy to sync in person if I misunderstand!